content/02.body.md

```diff
@@ -2,13 +2,71 @@
 
 ### Node degree
 
-![Degree figure](){#fig:degree width="100%"}
+Networks contain information about relationships between entities ("edges between nodes").
+One key aspect of a node (especially in unweighted networks) is its degree, the number of connections a node makes with other nodes in the network.
+Degree is an important metric for differentiating between nodes.
+Edge prediction (also known as "link prediction") methods tend to be heavily influenced by degree (Figure {@fig:feature-degree}).
+However, reliance on degree may not always be desirable.
+Firstly, degree may be a source of bias in networks.
```

Comment:
```suggestion
Firstly, degree may be influenced by bias in networks.
```
Is the idea that it's a source of bias or that bias influences degree which then leads to biased predictions? There are a lot of difficult questions about degree.

Reply:
I mean the latter. Degree is an outlet where bias (experimental, inspection, etc.) can manifest in the network. A downstream consequence of this is that predictions can be biased when compared to true connections.

I hesitate to say that degree is influenced by, "bias in networks," as it begs the question, "In what other ways besides degree can bias manifest in networks?" How is something like, "Firstly, bias in the data can distort node degree."